### PR TITLE
Remove Export Services for Geneva Release

### PR DESCRIPTION
**Pull request by michaelestrin**
Related to: https://github.com/edgexfoundry/edgex-go/issues/2218

Signed-off-by: Michael W. Estrin <me@michaelestrin.com>

### CODE REVIEW

**Review by lenny-goodell — DISMISSED** [review]
Code changes look appropriate. I didn't run BB tests to verify.

**Comment by cloudxxx8** [comment]
The PR for #295 will be opened after merging this one because there are some conflicts

File: bin/run.sh [hunk 1]
```diff
@@ -167,10 +158,6 @@ case ${option} in
 	supportNotificationTest	| tee $SUPPORT_NOTIFICATION_LOG_PATH
     ;;
   	-exc)
-    echo "Info: Initiating ExportClient Test"
-    exportClientTest | tee $EXPORTCLIENTLOGSPATH
-    ;;
-    -ss)
```
Comment (jinlinGuan) [inline bin/run.sh:173]: `-ss)` should be kept, but remove `-exc)`

**Review by jinlinGuan — CHANGES_REQUESTED** [review]

File: bin/run.sh (hunk 1 above)
Comment (michaelestrin) [reply bin/run.sh:173]: Great catch!

File: bin/run.sh [hunk 2]
```diff
@@ -166,11 +157,7 @@ case ${option} in
     echo "Info: Initiating SupportNotifications Test"
 	supportNotificationTest	| tee $SUPPORT_NOTIFICATION_LOG_PATH
     ;;
-  	-exc)
-    echo "Info: Initiating ExportClient Test"
-    exportClientTest | tee $EXPORTCLIENTLOGSPATH
-    ;;
-    -ss)
+  	-ss)
```
Comment (cloudxxx8) [inline bin/run.sh]: @michaelestrin please restore this line 160.  You are using tab instead of spaces

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Review by lenny-goodell — APPROVED** [review]
LGTM

**Review by jinlinGuan — APPROVED** [review]

**Comment by cloudxxx8** [comment]
@michaelestrin please help merge this PR.  Since there is no export-client in nightly build docker-compose file, we need this PR merge first and open another PR for #295
thanks

**Comment by michaelestrin** [comment]
> please help merge this PR. Since there is no export-client in nightly build docker-compose file, we need this PR merge first and open another PR for #295

@cloudxxx8 I don't understand what you're asking me to do.  This is a draft PR that is scheduled to be discussed later today in the Core WG meeting.  I currently don't know if/when this PR (and the others associated with removing the export services from master) will be merged.  You should proceed independently with the work for #295 and we'll just have to deal with the conflicts.

**Comment by cloudxxx8** [comment]
@michaelestrin , ok, we will open the PR for #295 first
@jinlinGuan please open the PR and remove the export client test trigger in your PR

**Pull request merged**